### ISSUE TITLE
Extend "not interested" with "don't recommend this channel"

### ISSUE THREAD

**Issue by Ketec** [yuliskov/SmartTubeNext#1438]
It's an option in the Youtube App itself - can we get an additional entry/shortcut for this?

Often blocking a single video does not work - youtube still keeps recommending the channel - or even recommending channels you are already subscribed to.
Currently, I have to use the youtube app just to get rid of some of the annoying channels - especially those crappy ones "funny x videos" "cool world stuff" just stealing videos and reuploading them as compilations.

**Comment by nephariouz** [comment]
Yes this! That is my only issue with this software. I have to spend far too much time telling it I'm not interested in 60+% of the videos it recommends because it keeps replacing those videos with more videos I'm not interested in from the same channel.

**Comment by Danny-Alexander** [comment]
This would be nice to have.
